schedule/storage/blktests_spvm.yaml

```diff
@@ -0,0 +1,10 @@
+name:           blktests_spvm
+description:    >
+    Executing blktests, the upstream testsuite
+vars:
+    DESKTOP: textmode
+    INSTALLONLY: 0
+    BOOT_HDD_IMAGE: 1
+schedule:
```

Comment:
missing installation/bootloader, please check https://openqa.suse.de/tests/4067255

Reply:
of course. my obvious overlook; I should have remembered that from the main_ltp.pm. sorry.